src/FixedDataTable.js

```diff
@@ -1026,25 +1026,12 @@ var FixedDataTable = createReactClass({
         );
 
         var previousColumnsWidth = 0;
-        for (i = 0; i < scrollableColumnIndex; ++i) {
+        for (i = 0; i < scrollableColumnIndex - 1; ++i) {
           column = columnInfo.bodyScrollableColumns[i];
           previousColumnsWidth += column.props.width;
         }
 
-        var availableScrollWidth = adjustedWidth - totalFixedColumnsWidth;
-        var selectedColumnWidth = columnInfo.bodyScrollableColumns[
-          scrollableColumnIndex
-        ].props.width;
-        var minAcceptableScrollPosition =
-          previousColumnsWidth + selectedColumnWidth - availableScrollWidth;
```

Comment:
No, we need to take into account fixed widths. We also need the logic to scroll backwards/forwards enough to show the column, not make it the first column